example/BUILD.bazel

```diff
@@ -0,0 +1,35 @@
+CMD = "$(CRANE_BIN) pull thesayyn/debian:oci $@ --format=oci --platform={platform}"
```

Comment:
nit: add some documentation in https://hub.docker.com/r/thesayyn/debian to how it was built & comment here pointing to that link